invenio_rdm_records/services/pids/providers/base.py

```diff
@@ -23,19 +24,21 @@ def __init__(self, username, password, url=None, **kwards):
 class BasePIDProvider:
     """Base Provider class."""
 
-    name = "base"
+    name = None
```

Comment:
None might fail validation, But your idea is that "No PID should be created with the base provider"?

Reply:
somehow force the user to put the name. Fix the tests.